### PR TITLE
Lastpass import tool

### PR DESCRIPTION
**Pull request by scantem**
Sorry this is attempt number 2 at a PR, end of the day here...

Firstly thanks very much for creating bitwarden-ruby, I looked at the official server but I wasn't happy with it's huge resource requirements.

This adds support for importing LastPass CSV export files. I've added support for importing the data using the existing LastPass folder structure so there's now a decrypt_data_with_master_password_key method in the user class so that I can decrypt & match existing folder names - if you would rather I kept the decrypt code in the lastpass_import.rb file I'm happy to change it.

If there are any problems or you have any concerns please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,24 @@ convert and import as many items as it can.
 This tool operates on the SQLite database directly (not through its REST API)
 so you can run it offline.
 
+### Lastpass Conversion
+
+Export everything from LastPass by going to your vault, "more options" and then "Export".
+It should will then export your details in a new window in CSV format, copy and paste this data into a csv file accessible from your bitwarden-ruby installation. Unfortunately due to limitations in LastPass export the "extra fields" and "attachments" data in the LastPass vault will not be converted.
```
Comment (jcs) [inline README.md]: Please hard-wrap these lines at <80 chars.

File: lib/user.rb [hunk 2]
```diff
@@ -43,6 +43,14 @@ def encrypt_data_with_master_password_key(data, mk)
     Bitwarden.encrypt(data, encKey[0, 32], encKey[32, 32])
   end
 
+  def decrypt_data_with_master_password_key(data, mk)
```
Comment (jcs) [inline lib/user.rb]: Please keep these methods alpha-sorted.

**Comment by scantem** [comment]
Sorry about the delay, all requested changes have now been made.

**Comment by jcs** [comment]
Thanks!

**Pull request merged**